editor/plugins/style_box_editor_plugin.cpp

```diff
@@ -58,7 +47,7 @@ void StyleBoxPreview::edit(const Ref<StyleBox> &p_stylebox) {
 	}
 	stylebox = p_stylebox;
 	if (p_stylebox.is_valid()) {
```

Comment:
We also seem to use `p_stylebox` and `stylebox` interchangeably in this section. Would be nice to stick to `stylebox` consistently after it is assigned the parameter value.